R/check-parent-syn.R

```diff
@@ -27,7 +27,7 @@ check_parent_syn <- function(manifest,
       )
     )
   }
-  behavior <- "Synapse IDs should follow the format 'syn' followed by digits"
+  behavior <- as.character("Synapse IDs should follow the format &#39;syn&#39; followed by digits") # nolint
```

Comment:
Is `as.character()` needed here? It's not present for the other behavior message you edited.

Reply:
Nope, I was throwing everything at that behavior to try to get it to behave nicely. After discovering the problem with the single quotes, I forgot to remove that bit. 🤦‍♀️